docs/features/models/SQuAD.ipynb

```diff
@@ -0,0 +1,679 @@
+   "outputs": [],
+   "source": [
+    "!pip install --q deeppavlov"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "vTBqvEtdpBpm"
+   },
+   "source": [
+    "Then make sure that all the required packages for the model are installed."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "RYx-1sDqpBpn"
+   },
+   "outputs": [],
+   "source": [
+    "!python -m deeppavlov install squad_bert"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "i6hPLgzvpBpn"
+   },
+   "source": [
+    "`squad_torch_bert` here is the name of the model's *config_file*. [What is a Config File?](http://docs.deeppavlov.ai/en/master/intro/configuration.html) \n",
+    "\n",
+    "Configuration file defines the model and describes its hyperparameters. To use another model, change the name of the *config_file* here and further.\n",
+    "The full list of the models with their config names can be found in the [table](#6.-Models-list)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "PEYknuKK9l_y"
+   },
+   "source": [
+    "\n",
+    "# 3. Use the model for prediction"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-j3tJy1c9l_1"
+   },
+   "source": [
+    "## 3.1 Predict using Python"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "cLnbUke-9l_3"
+   },
+   "source": [
+    "After [installing](#2.-Get-started-with-the-model) the model, build it from the config and predict."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "fnHw4a6C9l_4"
+   },
+   "outputs": [],
+   "source": [
+    "from deeppavlov import build_model\n",
+    "\n",
+    "model = build_model('squad_bert', download=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "ec572M3jpBpp"
+   },
+   "source": [
+    "**Input**: List[context, question]\n",
+    "\n",
+    "**Output**: List[answer, start_character, logit]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "colab": {
+     "base_uri": "https://localhost:8080/"
+    },
+    "id": "QFXXQf3Q9l_6",
+    "outputId": "c56dbc60-0730-47d6-835f-9317ac40a4b5"
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "[['a library for NLP and dialog systems'], [14], [200928.390625]]"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "model(['DeepPavlov is a library for NLP and dialog systems.'], ['What is DeepPavlov?'])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "RLeAxnlu9l_7"
+   },
+   "source": [
+    "## 3.2 Predict using CLI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "iE3j768spBps"
+   },
+   "source": [
+    "You can also get predictions in an interactive mode through CLI."
```

Comment:
"You can also get predictions in an interactive mode through CLI (Command Line Interface)."